scratch_script.py

```diff
@@ -0,0 +1,110 @@
+# scratch script to test various implementations in datta-lab-to-nwb
+from datta_lab_to_nwb.markowitz_gillis_nature_2023 import markowitz_gillis_nature_2023behaviorinterface
+from pynwb import NWBFile
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import colorcet as cc
+
+
+def run_conversion():
+    file_path = "/Volumes/T7/CatalystNeuro/NWB/Datta/dopamine-reinforces-spontaneous-behavior/dlight_raw_data/dlight_photometry_processed_full.parquet"
+    example_session = "2891f649-4fbd-4119-a807-b8ef507edfab"
+    interface = markowitz_gillis_nature_2023behaviorinterface.MarkowitzGillisNature2023BehaviorInterface(
+        file_path, example_session
+    )
+    metadata = interface.get_metadata()
+    nwbfile = NWBFile(**metadata["NWBFile"])
+    nwbfile = interface.run_conversion(nwbfile, metadata)
+    return nwbfile
+
+
+def reproduce_fig1d(nwbfile):
```

Comment:
Looks like this was moved to `utils` but it is still duplicated in this `scratch_script`; given the `convert_session.py` is now also being used, can you remove this scratch file altogether?